server/datastore/mysql/testing_utils.go

```diff
@@ -243,6 +243,11 @@ func createMySQLDSWithOptions(t testing.TB, opts *DatastoreTestOptions) *Datasto
 		strings.TrimPrefix(details.Name(), "github.com/fleetdm/fleet/v4/"), "/", "_",
 	)
 	cleanName = strings.ReplaceAll(cleanName, ".", "_")
+	if len(cleanName) > 60 {
+		// the later parts are more unique than the start, with the package names,
+		// so trim from the start.
+		cleanName = cleanName[len(cleanName)-60:]
+	}
```

Comment:
Could there be collision between two tests named the same way but on different packages?

Reply:
I think it's unlikely enough not to care about this?